code/modules/food_and_drink/snacks.dm

```diff
@@ -2186,6 +2202,29 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/soup)
 		else
 			src.UpdateOverlays(null, "dip")
 
+/obj/item/reagent_containers/food/snacks/churro
```

Comment:
Since this is just an exact copy paste of tortilla chip code you'd ideally want to make a parent type for these like `/obj/item/reagent_containers/food/snacks/dipping` and have both of them inherit from it

Reply:
this